apps/back/src/DataFixtures/AppFixtures.php

```diff
@@ -3,23 +3,37 @@
 namespace App\DataFixtures;
 
 use App\Entity\Courses;
+use App\Entity\User;
 use Doctrine\Bundle\FixturesBundle\Fixture;
 use Doctrine\Persistence\ObjectManager;
+use Symfony\Component\PasswordHasher\Hasher\UserPasswordHasherInterface;
 
 class AppFixtures extends Fixture
 {
-    public function __construct()
+    private UserPasswordHasherInterface $hasher;
+
+    public function __construct(UserPasswordHasherInterface $hasher)
     {
+        $this->hasher = $hasher;
     }
 
     public function load(ObjectManager $manager): void
     {
-        $courses = new Courses();
-        $courses->setName('my first courses');
-        $courses->setVideo('www.youtube/vidéo-courses.com');
 
+        $user = new User();
```

Comment:
Perhaps you could add email, username and password on the constructor ?

Reply:
I need user for hashPassword() function, i put email and username on constructor but i let password with setter.
what else can I do?

Reply:
It's all good with your modifications